blocks/embed/controller.php

```diff
@@ -30,7 +30,20 @@ public function getBlockTypeDescription()
 
     public function view()
     {
-        $this->set('info', Embed::create($this->source));
+        //$this->set('info', Embed::create($this->source));
+
+        try {
+            $info = Embed::create($this->source);
+
+        } catch (Exception $e) {
```

Comment:
This line catches nothing because `Concrete\Package\EmbedBlock\Block\Embed\Exception` class does not exist.

You should catch `\Exception` or `\Embed\Exceptions\InvalidUrlException` instead.

It's better to catch `InvalidUrlException` .
Catching all of exceptions is not a good design.